pages/translation.zh-hans.html

```diff
@@ -75,143 +75,156 @@ <h2 id="specs"><a href="#specs">W3C Specifications</a></h2>
-<p>We will check
-    the file before publication, and get back to you if changes are needed.</p>
+
+
+
+
+
+<section id="returning">
+<h3>发回翻译</h3>
+
+<p><strong>在将译文发给我们之前，请对照<a href="#checklist">翻译检查表</a>检查您的文件。</strong></p>
+<p>在通知我们翻译完成之前，请确保您的翻译是<a href="https://validator.w3.org/">有效的HTML5</a>文件并使用UTF-8字符编码。</p>
+  <p>如果您使用的是GitHub，请提交pull request。</p>
+<p>如果您不使用GitHub，您只需将修改了的文件作为附件单独发送到<strong><a href="mailto:public-i18n-translation@w3.org">public-i18n-translation@w3.org</a></strong>以在W3C官网上托管（我们将使用语言协商）。您无需发送未修改的任何文件。</p>
+<p>我们将在发布前检查文件，如果需要更改，我们会与您联系。</p>
 </section>
 
-<section>
-<h3 id="next"><a href="#next">What happens next?</a></h3>
 
-<p>After we received the translation, with any requested modifications, we'll post it to our GitHub staging site, where you can check it. When all is good, we'll publish to the main W3C site and send out an announcement.</p>
-<p>If you are translating more than one article, we'll usually wait for the whole batch to be ready before publishing to the W3C site and announcing.</p>
+
+
+
+<section id="next">
+<h3>接下来呢？</h3>
+
+<p>在我们收到（可能包含沟通过的修改的）翻译后，我们会将其发布到GitHub临时站点，您可以在其中检查这份文档。一切顺利后，我们将发布到W3C主站点并发出公告。</p>
+<p>如果您翻译的文章不止一篇，我们通常会等待整批文章都准备好，然后再发布到W3C国际化网站并发布新闻。</p>
 </section>
 
-<section>
-  <h3 id="updates"><a href="#updates">Updating pages</a></h3>
 
-  <p>If the English version of a page is updated, a note will be sent to the w3c-translators@w3.org list with information about what changed, and a request for the translator to update their translation.  </p>
-  <p>If the changes are only small additions, such as a new paragraph or a new short section, the English may be added to the translation while awaiting an update. The page will have a banner added to it that says it is out of date, and will point to the English original.</p>
-  <p>If the changes involve substantive or extensive changes, the translation will be removed until an updated version is provided.</p>
+
+
+
+<section id="updates">
+<h3><a href="#updates">更新页面</a></h3>
+
+  <p>如果页面的英文版有更新，我们会向w3c-translators@w3.org列表发送一封邮件，其中包含有关更改内容的信息，并请求翻译人员更新其翻译。</p>
+  <p>如果改动很小，例如只有一小段文字或者一个短章节，我们可能会在等待翻译更新时将英文原文添加到译文里。译文的页面将添加一个提示，表明页面已过时，并指向英文原版。</p>
+  <p>如果更改的内容较为广泛，则翻译将被删除，直到译文更新。</p>
 </section>
 </section>
 
 
   
-<section>
-    <h2><a name="contacts" id="contacts">Contacts</a></h2>
+<section id="contacts">
+<h2>联系我们</h2>
 
     <ul>
-        <li><a href="mailto:ishida@w3.org">Richard Ishida (ishida@w3.org)</a>, for translation of Internationalization Activity material.</li>
-        <li><a href="mailto:ivan@w3.org">Coralie Mercier (coralie@w3.org)</a>, for translation of other W3C material.</li>
+        <li><a href="mailto:ishida@w3.org">Richard Ishida (ishida@w3.org)</a>, 关于翻译国际化活动内容的问题</li>
+        <li><a href="mailto:ivan@w3.org">Coralie Mercier (coralie@w3.org)</a>, 关于翻译其他W3C文档的问题</li>
```

Comment:
Link points to Ivan's email address.  Needs to be changed to Coralie's.

Reply:
Fixed both the translation and the English version. Thanks.